packages/admin-ui/src/pages/packages/components/Info/ContainerList.tsx

```diff
@@ -27,31 +27,40 @@ export const ContainerList = ({ dnp }: { dnp: InstalledPackageData }) => {
 
   function onStartStop(container?: PackageContainer) {
     const dnpName = dnp.dnpName;
+    const isWifi = isWifiPackage();
     const serviceNames = container && [container.serviceName];
     const name = container
       ? [sn(dnpName), sn(container.serviceName)].join(" ")
       : sn(dnpName);
+    if (isWifi) {
+      confirm({
```

Comment:
You can mantain a linear control flow with only a function by turning `confirm` into a Promise
```ts
new Promise(resolve => {
  confirm({
    onClick: resolve
  })
})

```